tests/flags/README.md

```diff
@@ -0,0 +1,19 @@
+# Flag Tests
+
+Tests located in sub-directories of this one will be executed by passing the
+specified flags to Deno on the command line. Multiple flags are denoted by an
+`_` in the directory name.
+
+For example if a test was named `tests/flags/foo/test.ts` with a corresponding
+`tests/flags/foo/test.ts.out` the command line to Deno would be:
+
+```
+$ deno --foo tests/flags/foo/test.ts
```

Comment:
Mapping directories to flags not flexible enough - what happens when we want to test the output of `deno -e "console.log(1)"` ?

Other projects have solved this problem before... but I can't find any examples right now. 

I suggest modifying the output file to have some syntax... here's something off the top of my head:
```
command: deno -e "console.log('hello')"
output:
hello
```